joomla/joomla-cms
# Fix workflow notification plugin

### Summary of Changes
Resolves also  #29371.

Fix language strings.
Make sure that only users with access to com_messages get a notification.
Remove superfluous filters for users.


### Testing Instructions
You must have workflow enabled and at least one article.
Go to workflows, select transitions. 
For one transition i.e. published, set the sendMessage to on and add manager and administrator to receiver groups. Save and close.

Then go to articles.
Perform the transition for an article.
The transition must be performed and you must get a message that notification is sent.

Check that only users of the group administrator have got the message, but not users of the group manager (you can see this in table #__message).

Check the messages and texts.


### Expected result
as decribed


### Documentation Changes Required
no
